packages/reactive_image_picker/lib/src/selected_file_view.dart

```diff
@@ -3,7 +3,7 @@ import 'package:reactive_image_picker/src/image_file.dart';
 import 'package:reactive_image_picker/src/image_view.dart';
 import 'package:reactive_image_picker/src/video_view.dart';
 
-typedef OnChange = void Function(BuildContext context, SelectedFile file);
+typedef OnChange = void Function(BuildContext context, SelectedFile? file);
```

Comment:
simply allowing users to pass `null` to `_handleChange` will not allow users to add new files, rather it would replace all currently selected files